testing/opae-c/test_enum_c.cpp

```diff
@@ -368,7 +310,11 @@ TEST_P(enum_c_p, socket_id) {
   EXPECT_EQ(
       fpgaEnumerate(&filter_, 1, tokens_.data(), tokens_.size(), &num_matches_),
       FPGA_OK);
-  EXPECT_EQ(num_matches_, GetNumFpgaTypes() * 2);
+  if (device.socket_id == 0) {
+    EXPECT_EQ(num_matches_, GetNumFpgaTypes() * 2);
+  } else {
+    EXPECT_EQ(num_matches_, GetNumFpgaTypes());
```

Comment:
These checks will fail depending on the system. We need to check that num_matches_ is equal to the number of devices on the system that match the socket id.